content/general/concepts/qr-code/qr-code.md

```diff
@@ -0,0 +1,37 @@
+
+QR Codes do need to be square in shape, but there is room for some artistic styling. The squares and data dots can have rounded corners. The QR Code does not have to be black on white. The black can be changed to another color, but a higher contrast leads to more readability. A logo can be included in the center square. These changes make the QR Code visually unique and more suited to its use.
+
+## Reading a QR Code
+
+OK, but how does one read a QR Code? The good news is that one does not need a special laser bar code reader. Most camera apps in mobile devices contain the QR Code scanning software. Just open the camera app, focus on the QR Code, and a clickable link appears in the camera app.
+
+## Creating a QR Code
+
+Software is used to create the QR Code. The user enters data to be encoded and how the QR Code will be used. When the user is done entering the data, the software will create the QR Code. There are several QR Code generator websites. Some will generate basic QR Codes for free.
+
+Some QR Codes are static, and some are dynamic. A static QR Code has all the data encoded, and the data will not change. If a data entry error is made during QR Code creation, that error is permanent. A dynamic QR Code fixes the problem. A dynamic QR	Code will retrieve data from the internet, allowing the QR Code owner to edit the data as needed.
+
+## Uses for QR Codes
+
+QR Codes have many more uses than regular bar codes. A QR Codes hold more data, and they are easily readable by today’s mobile devices. The data can be just text, or instructions to open an app and populate fields. A QR Code can hold the URL for a web page or help log into a Wi-Fi network. QR Codes can show proof of purchase.  They are also used to track how many times the QR Code has been scanned. There are several other uses as well. 
+
+One of the most common uses for QR Codes is an URL for a web page. Instead of requiring a user to enter “codecademy.com/resources/docs/general” into a web browser, the URL is embedded into a QR Code. The user scans the QR Code and clicks the link. Many restaurants use QR Code URLs for on-line menus. Sporting events use QR Codes allowing spectators can get up to the minute stats. Charitable organizations use QR Codes directing doners to the donation web site.
+
+QR Codes are also used as entry passes. When purchasing a movie ticket online, the user will be sent a QR Code. The user retrieves the QR Code on a mobile device, and at the movie theater, the user has the QR Code scanned and is admitted.
+
+But QR Codes can also be used to point a user to a malicious web site, just like a phishing email. One should be careful about scanning and clicking on a QR Code. A QR Code passed out at a conference, enabling the attendees to log into the local Wi-Fi is going to be safe to scan and click. Just like emails, use good judgement before scanning and clicking.
```

Comment:
This paragraph is missing a counter-example of a QR code that would probably be unsafe to scan: A code posted on a hand-made flyer stuck to a lamppost, for example.